sdk/react/widgets/CastPlayPauseButtons.js

```diff
@@ -0,0 +1,352 @@
+  }
+
+  componentWillReceiveProps(nextProps) {
+    const { playing } = this.props;
+
+    if (nextProps.playing !== playing) {
+      this.state.playing = nextProps.playing;
+    }
+  }
+
+  componentWillUnmount() {
+    timerForSkipButtons.clearTimeout(this);
+  }
+
+  onPress() {
+    const { showButton, onPress } = this.props;
+    if (showButton) {
+      onPress(BUTTON_NAMES.PLAY_PAUSE);
+    } else {
+      onPress(BUTTON_NAMES.RESET_AUTOHIDE);
+    }
+  }
+
+  onSkipPress(isForward) {
+    const { onSeekPressed } = this.props;
+    timerForSkipButtons.clearTimeout(this);
+    const value = this.state.skipCount + (isForward ? 1 : -1);
+    this.setState({ skipCount: value }, () => timerForSkipButtons.setTimeout(
+      this,
+      'sendSummedSkip',
+      () => {
+        onSeekPressed(this.state.skipCount);
+        this.setState({ skipCount: 0 });
+      },
+      VALUES.DELAY_BETWEEN_SKIPS_MS,
+    ));
+  }
+
+  onSwitchPressed(isForward) {
+    const { onSwitchPressed } = this.props;
+
+    onSwitchPressed(isForward);
+  }
+
+  renderPlayPauseButton() {
+    const { loading, playing } = this.props;
+
+    let renderButton = null;
+    if (loading === true) {
+      renderButton = this.renderLoadingButton();
+    } else {
+      renderButton = playing ? this.renderButton(PAUSE) : this.renderButton(PLAY);
+    }
+    return renderButton;
+  }
+
+  renderLoadingButton() {
+    const { buttonWidth, buttonHeight, opacity } = this.props;
+    const accessible = true;
+    const sizeStyle = {
+      width: buttonWidth * 2,
+      height: buttonHeight * 2,
+    };
+
+    return (
+      <TouchableHighlight
+        accessible={accessible}
+        underlayColor="transparent"
+        activeOpacity={opacity}
+        importantForAccessibility="yes"
+        style={[sizeStyle, {
+          justifyContent: 'center',
+          alignItems: 'center',
+        }]}
+      >
+        <ActivityIndicator
+          size={60}
+          color="white"
+          style={[sizeStyle, {
+            justifyContent: 'center',
+            alignItems: 'center',
+          }]}
+        />
+      </TouchableHighlight>
+    );
+  }
+
+  renderButton(name) {
+    const {
+      fontSize, icons, buttonColor, opacity, buttonWidth, buttonHeight,
+    } = this.props;
+    const { playPause } = this.state;
+    const { animationOpacity, animationScale } = playPause;
+
+    const accessible = true;
+
+    const fontStyle = {
+      fontSize,
+      fontFamily: icons[name].fontFamily,
+    };
+    const finalOpacity = { opacity: animationOpacity };
+    const animate = { transform: [{ scale: animationScale }] };
+    const finalButtonColor = { color: !!buttonColor ? buttonColor : 'white' };
+    const sizeStyle = {
+      width: buttonWidth * 2,
+      height: buttonHeight * 2,
+    };
+    const label = AccessibilityUtils.createAccessibilityForPlayPauseButton(name);
+
+    return (
+      <TouchableHighlight
+        accessible={accessible}
```

Comment:
I suppose `accessible` declaration above is obsolete, you can write just 
```
<TouchableHighlight
  accessible
  accessibilityLabel={label}
  ...
```

Reply:
Done